builddefs/common_features.mk

```diff
@@ -500,7 +500,7 @@ ifeq ($(strip $(BACKLIGHT_ENABLE)), yes)
     endif
 endif
 
-VALID_WS2812_DRIVER_TYPES := bitbang pwm spi i2c
+VALID_WS2812_DRIVER_TYPES := bitbang pwm spi i2c pio
```

Comment:
Rather than introduce a new set of rather specific terminology, I think it would be best to inherit `vendor` that is used elsewhere.

Reply:
Done. Custom vendor drivers have a place in `chibios/drivers/vendor/RP/RP2040` as `ws2812_vendor.c` and `serial_vendor.c`